### DISCUSSION TITLE
How to tell which DWM (version of Spiral)

### DISCUSSION THREAD

**Discussion by methanoid** [SpiralLinux/SpiralLinux-project#418] [Q&A]
Looking for a way to detect which version of Spiral I am running.... (by way of a script).  Basically I love Spiral, but whenever I install it I script my own customizations. I'd like to maintain a master script (which does all the customizations I do to ALL installs) and then depending on the DWM detected, runs a set of customizations for that version (so for Plasma Spiral different stuff to Gnome or Cinnamon Spirals)

So, how to detect what version?

**Comment by geckolinux** [comment]
Hi there, one of the possible methods would be to check for this environment variable:
`echo $XDG_SESSION_DESKTOP`
Note that this command needs to be run as the user that is actually logged into the desktop, and not as `root`.
